models/fivetran_log__mar_table_history.sql

```diff
@@ -1,12 +1,12 @@
-with active_volume as (
+with incremental_mar as (
 
     select 
         *, 
-        {{ dbt.date_trunc('month', 'measured_at') }} as measured_month
+        {{ dbt.date_trunc('month', 'measured_date') }} as measured_month
 
-    from {{ ref('stg_fivetran_log__active_volume') }} 
+    from {{ ref('stg_fivetran_log__incremental_mar') }} 
 
-    where schema_name != 'fivetran_log' -- it's free! 
+    -- where schema_name != 'fivetran_log' -- it's free! 
```

Comment:
Nitpick here, but is there a reason we don't just delete this line instead of commenting it out?

Reply:
Good callout, this was me commenting out to test. It isn't needed anymore and can be removed.

Reply:
Removed